app/services/api/map_hesa_attributes/v01.rb

```diff
@@ -112,6 +138,50 @@ def course_attributes
 
         attributes
       end
+
+      def funding_attributes
+        ::Trainees::MapFundingFromDttpEntityId.call(funding_entity_id:)
+      end
+
+      def funding_entity_id
+        ::Hesa::CodeSets::BursaryLevels::MAPPING[params[:bursary_level]]
```

Comment:
The docs have this param as `funding_method` so we should probably use that.
```suggestion
        ::Hesa::CodeSets::BursaryLevels::MAPPING[params[:funding_method]]
```